packages/fast-components/src/slider/slider.styles.ts

```diff
@@ -0,0 +1,135 @@
+import { css } from "@microsoft/fast-element";
+import { disabledCursor, display } from "../styles";
+import { focusVisible } from "../styles/focus";
+import { SystemColors } from "../styles/system-colors";
+
+export const SliderStyles = css`
+    :host {
+        --thumb-size: calc(var(--height-number) * 0.5);
+        --thumb-translate: calc(var(--thumb-size) * 0.5);
+        --track-overhang: calc((var(--design-unit) / 2) * -1);
+    }
+    :host([hidden]) {
+        display: none;
+    }
```

Comment:
These should be able to drop under the host implementation on line 16. The display none behavior is inherit with the display helper.

Reply:
done and done